test-blocks/src/testblock.c

```diff
@@ -0,0 +1,78 @@
+#ifndef TESTBLOCK_URN
+    #define TESTBLOCK_URN "urn:mod-connector:testblock"
```

Comment:
the fallback here seems wrong to me, best to actually fail with `#error` instead, to make sure we dont misbuild things